src/tcgwars/logic/impl/gen8/BlueSkyStream.groovy

```diff
@@ -262,15 +276,18 @@ public enum BlueSkyStream implements LogicCardInfo {
         move "Return", {
           text "30 damage. You may draw cards until you have 6 cards in your hand."
           energyCost G
-          attackRequirement {}
           onAttack {
             damage 30
+            def count = 6 - my.hand.size()
+            if (count > 0
```

Comment:
Perhaps add a deck check here?

Reply:
`draw` uses `DrawCard` which properly handles no more cards in deck and broadcasts a message saying as such.